components/eam/src/dynamics/se/stepon.F90

```diff
@@ -513,6 +515,8 @@ subroutine stepon_run3(dtime, cam_out, phys_state, dyn_in, dyn_out)
    type (dyn_export_t), intent(inout) :: dyn_out ! Dynamics export container
    type (element_t), pointer :: elem(:)
    integer :: rc, i, j, k, p, ie, tl_f
+   integer :: pv_idx
```

Comment:
This does not appear to be used, please revert.

Reply:
removed